codalab/worker/main.py

```diff
@@ -190,6 +191,11 @@ def parse_args():
         default=1,
         help='The shared memory size of the run container in GB (defaults to 1).',
     )
+    parser.add_argument(
+        '--use-shared-cache',
```

Comment:
todo: remove this arg and just use a different branch to test the new dependency manager